bot.py

```diff
@@ -19,41 +19,39 @@
 
 import logging
 from datetime import datetime
-from random import randint
 
-from telegram import ParseMode, Message, Chat, InlineKeyboardMarkup, \
+from telegram import ParseMode, InlineKeyboardMarkup, \
     InlineKeyboardButton
 from telegram.ext import InlineQueryHandler, ChosenInlineResultHandler, \
     CommandHandler, MessageHandler, Filters, CallbackQueryHandler
 from telegram.ext.dispatcher import run_async
 
-from start_bot import start_bot
-from results import (add_call_bluff, add_choose_color, add_draw, add_gameinfo,
-                     add_no_game, add_not_started, add_other_cards, add_pass,
-                     add_card)
-from user_setting import UserSetting
-from utils import display_name, get_admin_ids
 import card as c
+import settings
+import simple_commands
+from actions import do_skip, do_play_card, do_draw, do_call_bluff, start_player_countdown
+from bot_config import WAITING_TIME, DEFAULT_GAMEMODE, MIN_PLAYERS
 from errors import (NoGameInChatError, LobbyClosedError, AlreadyJoinedError,
                     NotEnoughPlayersError, DeckEmptyError)
-from utils import send_async, answer_async, error, TIMEOUT
-from shared_vars import botan, gm, updater, dispatcher
 from internationalization import _, __, user_locale, game_locales
-import simple_commands
-import settings
-
-from simple_commands import help
+from results import (add_call_bluff, add_choose_color, add_draw, add_gameinfo,
+                     add_no_game, add_not_started, add_other_cards, add_pass,
+                     add_card, add_mode_classic, add_mode_fast, add_mode_wild)
+from shared_vars import botan, gm, updater, dispatcher
+from simple_commands import help_handler
+from start_bot import start_bot
+from utils import display_name
+from utils import send_async, answer_async, error, TIMEOUT, user_is_creator_or_admin, user_is_creator, game_is_running
 
-#import json
-#with open("config.json","r") as f:
-#    config = json.loads(f.read())
-#forbidden = config.get("black_list", None)
 
 logging.basicConfig(
     format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
-    level=logging.INFO)
+    level=logging.INFO
+    ,filename='log.log'
```

Comment:
I would prefer logging to stdout - if one wants to log to a file they can use the `>` operator